src/drivers/nuopc/ice_comp_nuopc.F90

```diff
@@ -20,7 +20,12 @@ module ice_comp_nuopc
   use shr_orb_mod            , only : shr_orb_decl, shr_orb_params, SHR_ORB_UNDEF_REAL, SHR_ORB_UNDEF_INT
   use shr_cal_mod            , only : shr_cal_noleap, shr_cal_gregorian, shr_cal_ymd2date
   use ice_shr_methods        , only : chkerr, state_setscalar, state_getscalar, state_diagnose, alarmInit
-  use ice_shr_methods        , only : set_component_logging, get_component_instance, state_flddebug
+#ifdef CESMCOUPLED
+  use nuopc_shr_methods      , only : set_component_logging, get_component_instance
+#else
+  use ice_shr_methods        , only : set_component_logging, get_component_instance
+#endif
```

Comment:
Nevermind. I guess these come from different places. Keep in mind that CESMCOUPLED is only for differences between NOAA UFS and CESM. Is that needed here?

Reply:
I spent all morning working on nuopc_shr_methods and couldn't figure out why it was working on everything but cice.   Yes this is a difference between UFS and CESM.